baselines/dom.generated.d.ts

```diff
@@ -3763,9 +3764,9 @@ interface HTMLCanvasElement extends HTMLElement {
       * Returns an object that provides methods and properties for drawing and manipulating images and graphics on a canvas element in a document. A context object includes information about colors, line widths, fonts, and other graphic parameters that can be drawn on a canvas.
       * @param contextId The identifier (ID) of the type of canvas to create. Internet Explorer 9 and Internet Explorer 10 support only a 2-D context using canvas.getContext("2d"); IE11 Preview also supports 3-D or WebGL context using canvas.getContext("experimental-webgl");
       */
-    getContext(contextId: "2d"): CanvasRenderingContext2D;
-    getContext(contextId: "experimental-webgl"): WebGLRenderingContext;
-    getContext(contextId: string, ...args: any[]): CanvasRenderingContext2D | WebGLRenderingContext;
+    getContext(contextId: "2d", contextAttributes?: {alpha?: boolean; willReadFrequently?: boolean; storage?: boolean; [attrib: string]: boolean | string}): CanvasRenderingContext2D;
```

Comment:
@mhegazy Sure, on it, although Canvas2DContextAttributes is probably a better name for it :)